ui/appui-react/src/appui-react/statusfields/SnapMode.tsx

```diff
@@ -68,6 +69,28 @@ const allSnapModeFieldEntries: SnapModeFieldEntry[] = [
     value: SnapMode.Bisector as number,
     icon: <SvgSnapsBisector />,
   },
+  // @ts-ignore Introduced in @itwin/core-frontend@5.0.0
+  ...(SnapMode.PerpendicularPoint === undefined
+    ? []
+    : [
+        {
+          labelKey: "snapModeField.perpendicularPoint",
+          // @ts-ignore Introduced in @itwin/core-frontend@5.0.0
+          value: SnapMode.PerpendicularPoint as number,
+          icon: <SvgPlaceholder />,
```

Comment:
Waiting for icons https://github.com/iTwin/appui/issues/1280#issuecomment-2886543654

Reply:
[Add icons](https://github.com/iTwin/appui/pull/1313/commits/2856cc2e76ba2e6ee6c71a60978f98b3eeef0688)